### DISCUSSION TITLE
Why would I need to provide a fill color for trail?

### DISCUSSION THREAD

**Discussion by longnguyen2004** [matteobruni/tsparticles#5103] [Q&A] [answered]
My use case is overlaying tsParticles on another element, so I need the background to be transparent. However, the trail config requires a fill color/image, which when I test it, fills the background with that color/image, which I don't want. Is there a way around this?

**Comment by matteobruni** [comment, marked as answer]
You can read more about it [here](https://github.com/matteobruni/tsparticles/issues/4882)

**Reply by longnguyen2004** [reply]
Wow didn't think there were so many duplicates, and the line workaround is brilliant, thanks!
